scenes/game/local_game.gd

```diff
@@ -2382,6 +2405,35 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			var close_amount = abs(performing_start - new_location)
 			local_conditions.fully_closed = close_amount == effect['amount']
 			_append_log("%s Close %s - Moved from %s to %s." % [performing_player.name, str(effect['amount']), str(previous_location), str(new_location)])
+		"copy_other_hit_effect":
+			var card = null
+			if performing_player == active_strike.initiator:
+				card = active_strike.initiator_card
+			else:
+				card = active_strike.defender_card
+			var hit_effects = get_all_effects_for_timing("hit", performing_player, card)
+
+			var effect_options = []
+			for possible_effect in hit_effects:
+				if possible_effect != effect:
```

Comment:
Do you need to have something in here to exclude the other copy of this boost?
I expect possible_effect != effect is doing a reference equality comparison check and not a deep comparison.

Reply:
Changed it to check the effect type instead; hopefully safe to assume this effect won't show up in an "and" or anything